package.json

```diff
@@ -1896,6 +1896,12 @@
                     "description": "%mssql.objectExplorer.groupBySchema%",
                     "default": false
                 },
+                "mssql.objectExplorer.connectionGroupsCollapsedOnStartup": {
+                    "type": "boolean",
+                    "description": "Connection Groups Collapsed on Startup",
```

Comment:
Please localize the string. Add the loc string to package.nls.json and refer it here by id. 
`%mssql.objectExplorer.connectionGroupsCollapsedOnStartupDescription%'